lnrpc/chainrpc/chainkit.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package chainrpc;
+
+option go_package = "github.com/lightningnetwork/lnd/lnrpc/chainrpc";
+
+// ChainKit is a service that can be used to get information from the
+// chain backend.
+service ChainKit {
+    // GetBlock returns a block given the corresponding block hash.
+    rpc GetBlock (GetBlockRequest) returns (GetBlockResponse);
+
+    /*
+    GetBestBlock returns the block hash and current height from the valid
+    most-work chain.
+     */
```

Comment:
nit: end of comment is indented by one space too much. Also, for the API generator we want all RPCs that have a corresponding CLI command to have the ```lncli: `<command>` ``` tag (see `lightning.proto` for examples).

Reply:
> end of comment is indented by one space too much.

Good eye!